test/plistToJson.test.ts

```diff
@@ -0,0 +1,96 @@
+// SPDX-FileCopyrightText: Copyright (c) 2021-2025 Objectionary.com
+// SPDX-License-Identifier: MIT
+
+import * as fs from "fs";
+import { plistToJson } from "../src/lib/plistToJson";
+
+jest.mock("fs");
+const mockedFs = jest.mocked(fs);
+
+const validPlistContent = `<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>name</key>
+    <string>Test Language</string>
+    <key>scopeName</key>
+    <string>source.test</string>
+    <key>fileTypes</key>
+    <array>
+        <string>test</string>
+    </array>
+    <key>patterns</key>
+    <array>
+        <dict>
+            <key>name</key>
+            <string>comment.line.test</string>
+            <key>match</key>
+            <string>#.*$</string>
+        </dict>
+    </array>
+</dict>
+</plist>`;
+
+const expectedJsonOutput = {
+  name: "Test Language",
+  scopeName: "source.test",
+  fileTypes: ["test"],
+  patterns: [
+    {
+      name: "comment.line.test",
+      match: "#.*$",
+    },
+  ],
+};
+
+describe("plistToJson tests", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should convert valid PLIST to JSON", () => {
+    mockedFs.readFileSync.mockReturnValue(validPlistContent);
+    const inputPath = "/test/input.tmLanguage";
+    const outputPath = "/test/output.json";
+
+    plistToJson(inputPath, outputPath);
+
+    expect(mockedFs.readFileSync).toHaveBeenCalledWith(inputPath, "utf-8");
+    expect(mockedFs.writeFileSync).toHaveBeenCalledWith(
+      outputPath,
+      JSON.stringify(expectedJsonOutput, null, 2)
+    );
+  });
+
+  it("should throw error for invalid PLIST content", () => {
+    mockedFs.readFileSync.mockReturnValue("This is not valid XML at all!");
+
```

Comment:
@svmukhin we usually don't tolerate empty lines in method bodies, here is why: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html